apps/src/openni_tracking.cpp

```diff
@@ -435,12 +435,12 @@ class OpenNISegmentTracking {
     result.width = cloud->width;
     result.height = cloud->height;
     result.is_dense = cloud->is_dense;
-    for (std::size_t i = 0; i < cloud->size(); i++) {
+    for (auto pt : *cloud) {
```

Comment:
```suggestion
    for (const auto& pt : *cloud) {
```

Reply:
@kunaltyagi thanks for pointing out, will make this change.